### PR TITLE
test: fix test on windows

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

file loader

##### Description of change

fix test on windows

### CODE REVIEW

**Comment by popomore** [comment]
Windows 过了

File: lib/utils/index.js [hunk 1]
```diff
@@ -36,11 +37,11 @@ module.exports = {
...
   // /path/to/app/controller/admin/config.js => controller.admin.config
-  getPathName(path, app) {
+  getPathName(fullpath, app) {
     const baseDir = app.loader.appInfo.baseDir;
-    return path
-      .replace(`${baseDir}/app/`, '')
-      .replace(/\//g, '.')
+    return fullpath
+      .replace(path.join(baseDir, 'app/'), '')
+      .replace(/\/|\\/g, '.')
```
Comment (dead-horse) [inline lib/utils/index.js]: `.replace(new RegExp(path.sep, 'g'), '.')`

Comment (atian25) [inline lib/utils/index.js]: 直接用 https://nodejs.org/api/path.html#path_path_normalize_path ?

**Comment by popomore** [comment]
@dead-horse 试了下不行，windows 的 sep 还需要转义

**Comment by popomore** [comment]
又改回来了

**Comment by popomore** [comment]
appveyor 又没回调，我合并了

**Pull request merged**